src/source/image_source.js

```diff
@@ -202,7 +208,7 @@ class ImageSource extends Evented implements Source {
             return this;
         }
         this.options.url = options.url;
-        this.load(options.coordinates, () => { this.texture = null; });
+        this.load(options.coordinates);
```

Comment:
Is the second parameter `successCallback` [here](https://github.com/mapbox/mapbox-gl-js/blob/f8df685266966c94b75b6cf9bd01501204304adb/src/source/image_source.js#L138) now unused? We could remove it if there's no other uses of it.

Reply:
Removed this in [7ee06b1](https://github.com/mapbox/mapbox-gl-js/pull/11564/commits/7ee06b183919e1791f87583f53c001926d2392d1)